doc/aspice_swe3_4/example/iceoryx_component/CMakeLists.txt

```diff
@@ -66,27 +48,17 @@ set(PROJECT_PREFIX "iceoryx_component")
 
 file(GLOB_RECURSE MODULETESTS_SRC "${CMAKE_CURRENT_SOURCE_DIR}/test/moduletests/*.cpp")
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${PROJECT_PREFIX}/test)
-
 set(TEST_LINK_LIBS
     GTest::gtest
     GTest::gmock
```

Comment:
It feels to specific somehow. Yes I could pack this into `iox_add_executable` but we would still add the `TEST_LINK_LIBS` since every test package as slightly different dependencies which are stored uniformly for all test variation (module, integration, stresstest etc).
For now I would leave it as it is and it could be changed later easily.